vars/sectionDeployToDeploymentTarget.groovy

```diff
@@ -109,7 +109,7 @@ def call(params) {
           stage("Functional Test - ${environmentDt} (staging slot)") {
             testEnv(deployer.getServiceUrl(environmentDt, "staging"), mergedTfOutput) {
               pcr.callAround("functionalTest:${environmentDt}") {
-                timeoutWithMsg(time: 40, unit: 'MINUTES', action: 'Functional Test') {
+                timeoutWithMsg(time: 120, unit: 'MINUTES', action: 'Functional Test') {
```

Comment:
We dont need this anymore with the new serenity tests and the cleaner AAT

Reply:
```suggestion
                timeoutWithMsg(time: 40, unit: 'MINUTES', action: 'Functional Test') {
```